cardinal/system/t_tokeniser_system_test.go

```diff
@@ -0,0 +1,81 @@
+package system
+
+import (
+	"testing"
+
+	"github.com/ArchetypalTech/TheOrugginTrail-ArgusWE/cardinal/enums"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewTokeniserSystem(t *testing.T) {
+	ts := NewTokeniserSystem()
+
+	assert.NotNil(t, ts.cmdLookup)
+	assert.NotNil(t, ts.dirLookup)
+	assert.NotNil(t, ts.dirObjLookup)
+	assert.NotNil(t, ts.objLookup)
+	assert.NotNil(t, ts.grammarLookup)
+	assert.NotNil(t, ts.responseLookup)
+
+	// Check if all predefined values are set correctly
+	assert.Equal(t, enums.ActionTypeGo, ts.cmdLookup["GO"])
+	assert.Equal(t, enums.ObjectTypeFootball, ts.objLookup["FOOTBALL"])
+	assert.Equal(t, enums.DirectionTypeNorth, ts.dirLookup["NORTH"])
+	assert.Equal(t, enums.DirObjectTypeDoor, ts.dirObjLookup["DOOR"])
+	assert.Equal(t, enums.GrammarTypeDefinitionArticle, ts.grammarLookup["THE"])
+	assert.ElementsMatch(t, []enums.ActionType{enums.ActionTypeBreak, enums.ActionTypeHit, enums.ActionTypeDamage}, ts.responseLookup[enums.ActionTypeKick])
+}
+
+func TestLookupFunctions(t *testing.T) {
+	ts := NewTokeniserSystem()
+
+	assert.Equal(t, enums.ObjectTypeFootball, ts.GetObjectType("FOOTBALL"))
+	assert.Equal(t, enums.ActionTypeGo, ts.GetActionType("GO"))
+	assert.Equal(t, enums.GrammarTypeDefinitionArticle, ts.GetGrammarType("THE"))
+	assert.Equal(t, enums.DirectionTypeNorth, ts.GetDirectionType("NORTH"))
+}
+
+func TestFishTokensKickTheBallToTheWindow(t *testing.T) {
+	ts := NewTokeniserSystem()
+
+	tokens := []string{"KICK", "THE", "BALL", "TO", "THE", "WINDOW"}
+	expectedVerb := enums.ActionTypeKick
+	expectedObj := enums.ObjectTypeFootball
```

Comment:
yes I see where you are going BUT we dont have an OBJECT type on its own, we have in this case : VRB: kick, DOBJ: football, IOBJ: window
thats what we always want, a VRB, a D(irect)OBJ and a I(ndirect)OBJ so we dont need an OBJECT in terms of the tokens

Reply:
same goes for all

Reply:
I followed the code of solidity for the fish tokens so the action "kick" is the VRB, the obj "ball" is the DOBJ and the dobj(direction object) "window" is the IOBJ 
on the enums objecttype is just for the type of object: ball, football, knife, etc
also the dirObjectType is for the direction object: window,  door, staits, etc

Reply:
![image](https://github.com/ArchetypalTech/TheOrugginTrail-ArgusWE/assets/72756999/5e80d04f-b5ae-4ad4-bdd1-e342abcac0e9)

In this working example with the process command tokens, it works and also the last blue line is the fish tokens result